res/openvpn-gui-res-zh-hant.rc

```diff
@@ -101,16 +102,16 @@ END
 /* Change Passphrase Dialog */
 ID_DLG_CHGPASS DIALOG 6, 18, 193, 82
 STYLE WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU | DS_CENTER
-CAPTION "OpenVPN - Change Passphrase"
+CAPTION "OpenVPN - 修改密碼"
```

Comment:
Well, you don't need to stick to it word-by-word, but have to consider the target context. 
I do use 更換座位 instead of 修改座位, but also 修改報告 instead of 更換報告. It depends on the action as described in the link you provided. Again, I do not see the differences for both terms when changing passphrases.